wa/workloads/googlemaps/__init__.py

```diff
@@ -0,0 +1,73 @@
+from wa import Parameter, ApkUiautoWorkload
+
+
+class GoogleMaps(ApkUiautoWorkload):
+
+    name = 'googlemaps'
+    description = '''
+    A workload to perform standard navigation tasks with Google Maps. This workload searches
+    for known locations, pans and zooms around the map, and follows driving directions
+    along a route.
+
+    To run the workload in offline mode, 'databases.tar' and 'files.tar' archives are required.
+    In order to generate these files, Google Maps should first be operated from an
+    Internet-connected environment, and a region around Cambridge, England should be downloaded
+    for offline use. This region must include the landmarks used in the UIAutomator program,
+    which include Cambridge train station and Corpus Christi college.
+
+    Following this, the files of interest can be found in the 'databases' and 'files'
+    subdirectories of the '/data/data/com.google.android.apps.maps/' directory. The contents of
+    these subdirectories can be archived into tarballs using commands such as
+    ``tar -cvf databases.tar -C /path/to/databases .``. These 'databases.tar' and 'files.tar' archives
+    should then be placed in the '~/.workload_automation/dependencies/googlemaps' directory on your
+    local machine, creating this if it does not already exist.
+
+    Known working APK version: 7.72.2
```

Comment:
I can't seem to find this version of google maps. Although I tested with 9.72.2 and it seems to work correctly, was this the version you meant?

Reply:
Oops, you're totally right it's 9.72.2.